spec/latest/core/index.html

```diff
@@ -1069,6 +1069,29 @@ <h4>Member assertions</h4>
     <p class="note">It's important to point out that the <code>subject</code>, <code>property</code>
       and <code>object</code> predicates are defined within the Hydra namespace and are not
       <a href="https://www.w3.org/TR/rdf-schema/"><code>rdf</code></a> terms.</p>
+
+    <p>It is also possible to use <i>memberAssertion</i> predicate on the API documentation level,
+      by attaching this predicate to subclasses of the <i>Collection</i>, like in the example below:</p>
+
+    <pre class="example nohighlight"
+         data-transform="updateExample"
+         title="Member assertion block describes strongly typed collection">
+      <!--
+      {
+        "@context": "http://www.w3.org/ns/hydra/context.jsonld",
```

Comment:
Adding this so that we can have a valid `api:` prefix as used below

```suggestion
        "@context": [
          "http://www.w3.org/ns/hydra/context.jsonld",
          {
            "api": "http://api.example.com/api/documentation#"
          }
        ],
```